src/styles/blocks/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --main-link-color: #00acdc;
+}
+
+.card__image {
+  padding: 32px 19px 0;
+  height: 134px;
+  width: 160px;
+}
+
+.card__description {
+  padding: 40px 16px 16px;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  margin: 0 0 4px;
+  line-height: 18px;
+}
+
+.card__product-code {
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__product-rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 0 24px;
+}
+
+.card__product-price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  text-align: center;
+  align-items: center;
+  margin: 0 0 16px;
+}
+
+.card__text {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__how-much {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__product-link-buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
when you move this property to `*`, delete it here and nothing will change

Reply:
I had some problems, I wrote it higher. Thanks)

Reply:
Explained it abow)